styles/styles.css

```diff
@@ -17,6 +17,12 @@
   padding-top: 0 !important;
 }
 
+.columns.table > .row:first-child {
+  text-transform: none !important;
```

Comment:
Do we need !important everywhere here ? I believe adding !important is not a very good practice unless its extremely unavoidable.

Reply:
yes i know !important is not good , since the same styles are coming from columns, no way we need to overwirite
<img width="762" alt="Screenshot 2023-07-05 at 07 25 23" src="https://github.com/adobecom/abpdocs/assets/9034600/03eefa4b-ed0a-4065-a9fa-70562f350870">